deployment/schema.json

```diff
@@ -121,8 +100,31 @@
         "description": "Never format with trailing commas."
       }]
     },
+    "jsonTrailingCommaDirectories": {
```

Comment:
Is this needed when one can write `.vscode/**.json`?

Reply:
True. I think I'll remove this and just make it so people have to specify explicit files for now. Ex. `.vscode/settings.json` (I don't want to bother adding a glob matcher atm)

Reply:
Oh, right, glob matching is in dprint, not the plugin.